e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js

```diff
@@ -73,40 +73,119 @@ test.describe('Time counductor operations', () => {
+        await expect(page.locator('.c-conductor__delta-button').first()).toContainText('00:30:23');
 
-        // Click time offset button
-        await page.locator('.c-conductor__delta-button >> text=00:30:00').click();
+        // Set end time offset
+        await setOffset(page, endOffset, false);
 
-        // Input start time offset
-        await page.fill('.pr-time-controls__secs', '23');
+        // Verify time was updated on preceding time offset button
+        await expect(page.locator('.c-conductor__delta-button').nth(1)).toContainText('00:00:31');
+    });
 
-        // Click the check button
-        await page.locator('.icon-check').click();
+    /**
+     * Verify that start and end offsets are preserved when switching between
+     * fixed timespan and real-time mode.
+     */
+    test('preserve offsets when switching between fixed and real-time mode', async ({ page }) => {
+        const startOffset = {
+            secs: '23'
+        };
 
-        // Verify time was updated on time offset button
-        await expect(page.locator('.c-conductor__delta-button').first()).toContainText('00:30:23');
+        const endOffset = {
+            secs: '00'
+        };
 
-        // Click time offset set preceding now button
-        await page.locator('.c-conductor__delta-button >> text=00:00:30').click();
+        //Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
 
-        // Input preceding time offset
-        await page.fill('.pr-time-controls__secs', '31');
+        // Switch to real-time mode
+        setTimeConductorMode(page, false);
 
-        // Click the check buttons
-        await page.locator('.icon-check').click();
+        // Set start time offset
+        await setOffset(page, startOffset, true);
 
-        // Verify time was updated on preceding time offset button
-        await expect(page.locator('.c-conductor__delta-button').nth(1)).toContainText('00:00:31');
+        // Set end time offset
+        await setOffset(page, endOffset, false);
+
+        // Switch to fixed timespan mode
+        setTimeConductorMode(page, true);
+
+        // Switch back to real-time mode
+        setTimeConductorMode(page, false);
+
+        // Verify updated start time offset persists after mode switch
+        await expect(page.locator('.c-conductor__delta-button').first()).toContainText('00:30:23');
+
+        // Verify updated end time offset persists after mode switch
+        await expect(page.locator('.c-conductor__delta-button').nth(1)).toContainText('00:00:00');
     });
 });
+
+/**
+ * @typedef {Object} OffsetValues
+ * @property {string | undefined} hours
+ * @property {string | undefined} mins
+ * @property {string | undefined} secs
+ */
+
+/**
+ * @param {import('@playwright/test').Page} page
+ * @param {OffsetValues} offset
+ * @param {boolean} isStartOffset
+ */
+async function setOffset(page, {hours, mins, secs}, isStartOffset = true) {
```

Comment:
I would update this function name to reference the domain object in some way in case someone decides to refactor this out of this test